app/.env.sample

```diff
@@ -1,6 +1,7 @@
 OCA_URL=https://raw.githubusercontent.com/bcgov/aries-oca-bundles/main/
 MEDIATOR_URL=https://f326-207-194-65-204.ngrok.io?c_i=eyJAdHlwZSI6ICJodHRwczovL2RpZGNvbW0ub3JnL2Nvbm5lY3Rpb25zLzEuMC9pbnZpdGF0aW9uIiwgIkBpZCI6ICI2MjQ0ZThiNS0wNWYzLTRhYWItYjM1Yy1lYWVlMWNmZTAyM2MiLCAicmVjaXBpZW50S2V5cyI6IFsiQ3lqM1BHRUJzQ3RyUGFtTTQyRngza3BlYmR2QWdNd1lGejlFS3RmNnlUN3giXSwgImxhYmVsIjogIk1lZGlhdG9yIiwgInNlcnZpY2VFbmRwb2ludCI6ICJodHRwczovL2YzMjYtMjA3LTE5NC02NS0yMDQubmdyb2suaW8ifQ==
```

Comment:
I know we spoke about this in a meeting, but is there a docs page we can store working copies of the `.env`?

If there is I can add the link as part of the PR

Reply:
I think we agreed this will just be something you ask for in RC and one of us will send it - no?